webapp/src/components/sidebar_header/index.js

```diff
@@ -1,6 +1,6 @@
 import {connect} from 'react-redux';
 
-import SidebarHeader from './sidebar_header.jsx';
+import SidebarHeader from './sidebar_header';
 
 function mapStateToProps(state) {
     const members = state.entities.teams.myMembers || {};
```

Comment:
@Willy-Wakam What are your thoughts on typing this file to `index.ts` as well? We can use this type for the `state` parameter:

```ts
import {GlobalState} from 'mattermost-redux/types/store';

function mapStateToProps(state: GlobalState) {
```